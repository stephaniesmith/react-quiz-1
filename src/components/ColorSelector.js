import React from 'react';
import PropTypes from 'prop-types';

const ColorSelector = ({ onChange }) => {
  return (
    <div>
      <label htmlFor="color">Font Color</label>
      <input type="color" name="color" onChange={onChange}/>
      <label htmlFor="backgroundColor">Background Color</label>
      <input type="color" name="backgroundColor" onChange={onChange}/>
    </div>
  );
};

ColorSelector.propTypes = {
  onChange: PropTypes.func.isRequired
};

export default ColorSelector;
