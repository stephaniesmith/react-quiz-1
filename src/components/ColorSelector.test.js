import React from 'react';
import { shallow } from 'enzyme';
import ColorSelector from './ColorSelector';

describe('ColorSelector', () => {
  it('renders ColorSelector', () => {
    const wrapper = shallow(<ColorSelector />);
    expect(wrapper).toMatchSnapshot();
  });
});
