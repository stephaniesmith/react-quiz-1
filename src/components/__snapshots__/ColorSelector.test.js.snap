// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ColorSelector renders ColorSelector 1`] = `
<div>
  <label
    htmlFor="color"
  >
    Font Color
  </label>
  <input
    name="color"
    type="color"
  />
  <label
    htmlFor="backgroundColor"
  >
    Background Color
  </label>
  <input
    name="backgroundColor"
    type="color"
  />
</div>
`;
